.github/resources/manifests/argo/kustomization.yaml

```diff
@@ -5,13 +5,13 @@ resources:
 - ../../../../manifests/kustomize/env/platform-agnostic
 
 images:
-- name: gcr.io/ml-pipeline/api-server
+- name: ghcr.io/kubeflow/kfp-api-server
```

Comment:
tests are failing due to [this](d2ddb2ed1c2afa64fd6014c95190416ff5cdd621) change, I falsely assumed things were working but the old images were being used 🤦🏾 , so I'm guessing that's why the server is crashing

I think the issue is probably [here](https://github.com/kubeflow/pipelines/blob/master/backend/Dockerfile#L57) 

changing `.[].file` to `.pipelines[].file` should fix it, since the format of the samples json is now different 

if it still fails let me know, this one probably falls on me to fix

Reply:
@HumairAK yup, my other PR adds logging to the failing pods (I'll shift that over here) and the log message is this:

```
I0129 02:20:31.147561      13 client_manager.go:216] Client manager initialized successfully
F0129 02:20:31.147947      13 main.go:82] Failed to load samples. Err: failed to load sample [Tutorial] Data passing in python components. Error: open /samples/tutorials/Data passing in python components/Data passing in python components - Files.py.yaml: no such file or directory

goroutine 1 [running]:
github.com/golang/glog.Fatalf(...)
	/go/pkg/mod/github.com/golang/glog@v1.2.0/glog.go:690
main.main()
	/go/src/github.com/kubeflow/pipelines/backend/src/apiserver/main.go:82 +0x2a9

SIGABRT: abort
PC=0x4098ae m=0 sigcode=18446744073709551610
```